### PR TITLE
Signup v2

### PR DESCRIPTION
**Pull request by Roshannair24**
updated navbar
signuoresp

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]

File: cart.html [hunk 1]
```diff
@@ -51,6 +51,7 @@ <h1>Iconic</h1>
             <i class="bx bx-cart"></i>
           </div>
         </a>
+        <i class="bx bx-menu"></i>
```
Comment (vaibhav18matere) [inline cart.html:54]: you can use better and descriptive class names instead of bx-cart

File: css/styles.css [hunk 2]
```diff
@@ -98,6 +102,29 @@ nav {
...
+.banner a {
+  background-color: rgba(23, 108, 184, 0);
+
+  color: var(--ascent-col);
+
+  position: relative;
+
+  display: inline-flex;
+
+  bottom: -15rem;
```
Comment (vaibhav18matere) [inline css/styles.css:114]: remove blank spaces

File: css/styles.css [hunk 3]
```diff
@@ -107,7 +134,7 @@ nav {
 
   display: flex;
 
-gap: 5px;
+  gap: 5px;
   /* justify-content: space-between;  */
 }
```
Comment (vaibhav18matere) [inline css/styles.css:140]: remove commented code

File: css/styles.css [hunk 4]
```diff
@@ -505,16 +532,148 @@ background-color: red;
...
+    display: flex;
+    flex-direction: column;
+  }
+
+  .cardsCollection {
+    display: flex;
+
+    flex-direction: column;
+
+    justify-content: center;
```
Comment (vaibhav18matere) [inline css/styles.css:560]: you can use separate class for this

**Review by vaibhav18matere — APPROVED** [review]

**Pull request merged**